src/psyclone/psyir/nodes/node.py

```diff
@@ -265,6 +265,7 @@ def dag(self, file_name='dag', file_format='svg'):
         try:
             graph = gv.Digraph(format=file_format)
         except ValueError:
+            from psyclone.psyGen import GenerationError
```

Comment:
pylint complains that GenerationError is being re-imported here (probably resulting from a merge)

Reply:
Yes, indeed - it was silently merged I believe, so I didn't rerun pylint on it.